src/main.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+int
+main(void)
+{
+    printf("Hello World\n");
```

Comment:
Acho que esse hello word ta ruim
```suggestion
    printf("Hello World!\n");
```